src/styles/_catalog.scss

```diff
@@ -1,22 +1,23 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  gap: $catalog-gap;
+  justify-content: center;
+  grid-template-columns: repeat(1, $card-width);
 
-  max-width: 944px;
-  margin: 0 auto;
+  @each $width, $col-number in $breakpoints {
+    @media (min-width: $width) {
+      grid-template-columns: repeat($col-number, $card-width);
+    }
+  }
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
     margin-bottom: 48px;
   }
```

Comment:
do u need it?

Reply:
Initially, I wanted to have a bigger distance between rows, but you're totally right. Cause, even in that case, I could have used gap with 2 values:facepalm: Fixed!